app/templates/contribute/ambassador.jade

```diff
@@ -30,7 +30,7 @@ block content
   
       h4(data-i18n="contribute.how_to_join") How to Join
       p
-        a(href="https://discourse.codecombat.com/", data-i18n="community.social_discource")
+        a(href="https://discourse.codecombat.com/c/ambassador/6", data-i18n="community.social_discource")
```

Comment:
Thoughts on changing it to a list of the following? Includes the link to wiki, slack channel, and the ambassador category. However, I'm not sure what or how the i18n system works, so I'm reluctant to push these changes unless `data-i18n` tags are updated on all locale files in GitHub.
```jade
ul
        li
          a(href="https://github.com/codecombat/codecombat/wiki/Ambassador-Home", data-i18n="") Read the documentation.
        li
          a(href="https://coco-slack-invite.herokuapp.com/", data-i18n="") Find us in our public Slack channel.
        li
          a(href="https://discourse.codecombat.com/c/ambassador/6", data-i18n="") Help others in the Ambassador category.
```

Reply:
@AndrewJakubowicz care to take a look at this one?